core/src/main/java/hudson/TcpSlaveAgentListener.java

```diff
@@ -153,6 +154,13 @@ public String getAgentProtocolNames() {
         return StringUtils.join(Jenkins.getInstance().getAgentProtocols(), ", ");
     }
 
+    /**
+     * Gets Remoting minimum supported version to prevent unsupported agents from connecting
+     */
```

Comment:
```suggestion
     * @since TODO 
     */
```

Reply:
Thanks for pointing that out, I've updated that part as you suggested.

Reply:
Actually, FYI, this must stay as `TODO` until being merged. We generally replace TODO by batches after the fact because a submitter cannot know in advance which version this will be merged in.

Reply:
I tried to commit back to TODO, but apparently from the UI it does not let me do it. @longngn can you please file a followup fix this week @longngn ? 

This is not such a big deal, so I'm going to merge this while it's ready for it.

Thanks!